src/main/groovy/com/deploygate/gradle/plugins/tasks/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.deploygate.gradle.plugins.tasks;
```

Comment:
this file end with `.java`.
Isn't it `.groovy` ?

Reply:
@satsukies 
Oh sorry, I haven't updated the development notes. 
As of https://github.com/DeployGate/gradle-deploygate-plugin/pull/144 and https://github.com/DeployGate/gradle-deploygate-plugin/pull/151, we are currently migrating from Groovy to Java because of the maintainability.

Reply:
> because of the maintainability.

I understand, sounds good!